arangod/RestServer/LanguageCheckFeature.cpp

```diff
@@ -170,40 +171,45 @@ LanguageCheckFeature::LanguageCheckFeature(Server& server)
 }
 
 void LanguageCheckFeature::start() {
+  using namespace arangodb::basics;
+
   auto& feature = server().getFeature<LanguageFeature>();
-  auto currDefaultLang = feature.getDefaultLanguage();
-  auto currIcuLang = feature.getIcuLanguage();
-  auto currLangType = feature.getLanguageType();
+  std::string_view currLang;
+  LanguageType currLangType;
+  std::tie(currLang, currLangType) = feature.getLanguage();
```

Comment:
```suggestion
  auto [currLang, currLangType] = feature.getLanguage();
```

Also now, you save temporary std::string to std::string_view, it looks like a bug

Reply:
Good point